commit fbe7823623a8c02759afdfb521709f4fa216849a
Author: Chris Morgan <macromorgan@hotmail.com>
Date:   Mon Feb 12 12:49:49 2024 -0600

    dt-bindings: arm: rockchip: Add Powkiddy RGB10MAX3
    
    The Powkiddy RGB10MAX3 is a handheld gaming device made by Powkiddy and
    powered by the Rockchip RK3566 SoC.
    
    Signed-off-by: Chris Morgan <macromorgan@hotmail.com>
    Acked-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
    Link: https://lore.kernel.org/r/20240212184950.52210-7-macroalpha82@gmail.com
    Signed-off-by: Heiko Stuebner <heiko@sntech.de>

diff --git a/Documentation/devicetree/bindings/arm/rockchip.yaml b/Documentation/devicetree/bindings/arm/rockchip.yaml
index 433a25f4e744..ca2432e803b3 100644
--- a/Documentation/devicetree/bindings/arm/rockchip.yaml
+++ b/Documentation/devicetree/bindings/arm/rockchip.yaml
@@ -694,6 +694,7 @@ properties:
       - description: Powkiddy RK3566 Handheld Gaming Console
         items:
           - enum:
+              - powkiddy,rgb10max3
               - powkiddy,rgb30
               - powkiddy,rk2023
               - powkiddy,x55
